docs(config) : Add number type to watchoption (#3676)
webpack/webpack.js.org: src/content/configuration/watch.md

};
```

> __webpack.config.js__

Reviewer: this example without explanation doesn't make much sense to me, do you want to explain what happens when you pass in a number?

};
```

> When number is provided instead of an object, `watchOptions.aggregateTimeout` is set to the argument passed.

Reviewer: This is uncommon to webpack docs (saying doing this instead of that), usually we just say `set to x and get these results... __example`
```suggestion
Providing a number to the `watchOptions` sets `watchOptions.aggregateTimeout` to the given number.
```

This way its shorter and clearer to the reader
